app/models/grda_warehouse/hud/project.rb

```diff
@@ -573,6 +573,14 @@ def project_type_overridden_as_ph?
 
     alias_attribute :name, :ProjectName
 
+    def confidential?
+      super || organization&.confidential?
```

Comment:
Does `super` delegate to the active record base class?  I like this if it does.

Reply:
Yep it does